service/metrics/requirements.txt

```diff
@@ -0,0 +1,4 @@
+pydantic==2.6.0
```

Comment:
Shy is this requirement file here?

Reply:
Removed the pydantic dependency from this file, indeed its not needed for the service.
Is this what you meant?